src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs

```diff
@@ -165,8 +165,18 @@ impl InstanceModel {
         let on_data_received              = Rc::new(on_data_received);
         let set_size                      = get_method(&object.as_ref(),method::SET_SIZE).ok();
         let set_size                      = Rc::new(set_size);
+        let on_hide                       = get_method(&object.as_ref(), method::ON_HIDE).ok();
+        let on_hide                       = Rc::new(on_hide);
         let object                        = Rc::new(object);
-        Ok(InstanceModel{object,on_data_received,set_size,root_node,logger,preprocessor_change})
+        Ok(InstanceModel {
+            object,
+            on_data_received,
+            set_size,
+            on_hide,
+            root_node,
+            logger,
+            preprocessor_change
+        })
```

Comment:
According to our guidelines, you should rather wrap line, instead putting all fields in a separate line. Look for SceneData `new` constructor and see an example there.